service-workers/service-worker/resources/import-scripts-get.py

```diff
@@ -0,0 +1,6 @@
+def main(req, res):
+    return ([
+        ('Cache-Control', 'no-cache, must-revalidate'),
+        ('Pragma', 'no-cache'),
+        ('Content-Type', 'application/javascript')],
+        req.GET['output'] + ' = "%s";\n' % req.GET['msg'])
```

Comment:
Style nit: please use string interpolation twice, rather than both concatenation and interpolation. (Possibly also move the output out of the return statement and into a local variable for readability.)

Reply:
Done